docs/third-party-developers/extensibility/data-store/cart.md

```diff
@@ -44,6 +67,365 @@ To utilize this store you will import the `CART_STORE_KEY` in any module referen
+
+#### _Example_ <!-- omit in toc -->
+
+```js
+const { dispatch } = useDispatch( CART_STORE_KEY );
+dispatch( receiveApplyingCoupon( couponCode ) );
+```
+
+### receiveRemovingCoupon
+
+This action returns an action object used to track when a coupon is removing.
+
+#### _Parameters_ <!-- omit in toc -->
+
+-   `couponCode: string`: The code of the coupon being removed.
+
+#### _Returns_ <!-- omit in toc -->
+
+-   `Object`: The action object.
+
+#### _Example_ <!-- omit in toc -->
+
+```js
+const { dispatch } = useDispatch( CART_STORE_KEY );
+dispatch( receiveRemovingCoupon( couponCode ) );
+```
+
+### receiveCartItem
+
+This action is used to update a specific item in the cart.
+
+#### _Parameters_ <!-- omit in toc -->
+
+-   `CartResponseItem: `:  A cart item API response.
+
+#### _Returns_ <!-- omit in toc -->
+
+-   `Object`: The action object.
+
+#### _Example_ <!-- omit in toc -->
+
+```js
+const { dispatch } = useDispatch( CART_STORE_KEY );
+dispatch( receiveCartItem( CartResponseItem ) );
+```
+
+### itemIsPendingQuantity
+
+This action returns an action object to indicate if the specified cart item quantity is being updated.
+
+#### _Parameters_ <!-- omit in toc -->
+
+-   `cartItemKey: string`: The key of the cart item.
+-   `isPending: boolean`: The new status of whether the cart item quantity is being updated. Defaults to `true`.
+
+#### _Returns_ <!-- omit in toc -->
+
+-   `Object`: The action object.
+
+#### _Example_ <!-- omit in toc -->
+
+```js
+const { dispatch } = useDispatch( CART_STORE_KEY );
+dispatch( itemIsPendingQuantity( cartItemKey, isPending ) );
+```
+
+### itemIsPendingDelete
+
+This action returns an action object to indicate if the specified cart item is being deleted.
+
+#### _Parameters_ <!-- omit in toc -->
+
+-   `cartItemKey: string`: The key of the cart item.
+-   `isPending: boolean`: The new status of whether the cart item is being deleted. Defaults to `true`.
+
+#### _Returns_ <!-- omit in toc -->
+
+-   `Object`: The action object.
+
+#### _Example_ <!-- omit in toc -->
+
+```js
+const { dispatch } = useDispatch( CART_STORE_KEY );
+dispatch( itemIsPendingDelete( cartItemKey, isPending ) );
+```
+
+### setIsCartDataStale
+
+This action returns an action object to indicate if the cart data is stale.
+
+#### _Parameters_ <!-- omit in toc -->
+
+-   `isCartDataStale: boolean`: Flag to mark cart data as stale; true if `lastCartUpdate` timestamp is newer than the one in wcSettings. Defaults to `true`.
+
+#### _Returns_ <!-- omit in toc -->
+
+-   `Object`: The action object.
+
+#### _Example_ <!-- omit in toc -->
+
+```js
+const { dispatch } = useDispatch( CART_STORE_KEY );
+dispatch( setIsCartDataStale( isCartDataStale ) );
+```
+
+### updatingCustomerData
+
+This action returns an action object to indicate if the customer data is being updated.
+
+#### _Parameters_ <!-- omit in toc -->
+
+-   `isResolving: boolean`: The new status of whether the customer data is being updated.
```

Comment:
"The new status" sounds a bit strange. For consistency-reasons, I'd use the similar wording for all boolean parameters and returns. Here are two examples of other docs:

https://github.com/woocommerce/woocommerce-blocks/blob/6960ae656112b5a5249673db2aaa80e13c4b94c7/docs/third-party-developers/extensibility/checkout-block/available-filters/coupons.md?plain=1#L84

https://github.com/woocommerce/woocommerce-blocks/blob/6960ae656112b5a5249673db2aaa80e13c4b94c7/docs/third-party-developers/extensibility/checkout-block/available-filters/coupons.md?plain=1#L151